lib/mmbooking_Clone/user/visitor.ex

```diff
@@ -28,6 +28,6 @@ defmodule Mmbooking_Clone.User.Visitor do
   def changeset(visitor, attrs) do
     visitor
     |> cast(attrs, [:id, :email_id, :first_name, :last_name, :dob, :country, :city, :have_you_visited_inner_chamber, :last_date_of_visit, :preferred_date, :alternate_date_of_visit, :place_of_stay, :arrival_date, :departure_date, :notes])
-    |> validate_required([:email_id, :first_name, :last_name, :dob, :country, :city, :have_you_visited_inner_chamber,  :preferred_date, :alternate_date_of_visit, :place_of_stay, :arrival_date, :departure_date, :notes])
```

Comment:
Why you removed fields from validate required?

Reply:
For the Add Visitor form, I need to insert only upto 6 to 7 fields, I don't need to insert other fields so i removed other fields that are not required.

Reply:
What about new visitor registration. Is this same changeset you using for new registration right?

Reply:
But there, we are only saving into the database only after every field is entered. We are saving only on the verify details page only.

Reply:
@sakthibalan15 tole me to use different changeset